src/routers/user.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+
+const router = express.Router();
+
+const userController = require('../controllers/user');
+
+// GET and POST routes for /
+router.route('/').get(userController.getAllUsers).post(userController.addUser);
```

Comment:
Can you ensure that `.get` and `.post` show on new lines? For better readability.